spec/acceptance/vhost_spec.rb

```diff
@@ -1330,6 +1330,13 @@ class { 'apache': }
     describe 'fastcgi' do
       it 'applies cleanly' do
         pp = <<-EOS
+          if ($::operatingsystem == 'Ubuntu') {
```

Comment:
Since this step is only applicable to Ubuntu versions >10.04, should there be an additional conditional check here to only add multiverse on versions > 10.04?